tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java

```diff
@@ -44,6 +47,21 @@
  * Utilities for Trogdor TaskWorkers.
  */
 public final class WorkerUtils {
+
+    private static final List<String> CLIENT_SECURITY_KEYS = Arrays.asList(
+        AdminClientConfig.SECURITY_PROTOCOL_CONFIG,
+        SaslConfigs.SASL_MECHANISM,
+        SaslConfigs.SASL_JAAS_CONFIG,
+        SaslConfigs.SASL_KERBEROS_SERVICE_NAME,
+        SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG,
+        SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG,
+        SslConfigs.SSL_KEY_PASSWORD_CONFIG,
+        SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG,
+        SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG,
+        SslConfigs.SSL_PROTOCOL_CONFIG,
+        SslConfigs.SSL_ENDPOINT_IDENTIFICATION_ALGORITHM_CONFIG
```

Comment:
Can you not use `.withClientSslSupport().withClientSaslSupport()`?

Reply:
I did not know about those methods. Let me look. Thanks!